python/mxnet/kvstore.py

```diff
@@ -120,8 +120,15 @@ def init(self, key, value):
         [ 2.  2.  2.]]
 
         >>> # init a list of key-value pairs
-        >>> keys = [5, 7, 9]
+        >>> keys = ['5', '7', '9']
         >>> kv.init(keys, [mx.nd.ones(shape)]*len(keys))
+
+        >>> # init a row_sparse value
+        >>> kv.init('4', mx.nd.ones(shape).tostype('row_sparse'))
+        >>> b = mx.nd.sparse.zeros('row_sparse', shape)
+        >>> kv.row_sparse_pull('4', row_ids=mx.nd.array([0, 1]), out=b)
```

Comment:
Without giving the dtype I get the error: "Unknown type enum 0". Would it make sense to pull just one of the rows out of the two as the example ?

Reply:
`Unknown type enum 0 ` -> this is fixed in this PR if you checkout my code. (see changes in kvstore_local.h)
In this example, I'm showing all the rows initialized so I gave all row_ids. In the example of `row_sparse_pull`, there's an example of pulling a subset of the rows.